windows/portaudio/JackPortAudioDriver.cpp

```diff
@@ -196,8 +196,12 @@ int JackPortAudioDriver::Open(jack_nframes_t buffer_size,
     }
 
     // Generic JackAudioDriver Open
+    char capture_driver_name[JACK_CLIENT_NAME_SIZE];
+    char playback_driver_name[JACK_CLIENT_NAME_SIZE];
+    snprintf(capture_driver_name, sizeof(capture_driver_name), "%s", capture_driver_uid);
+    snprintf(playback_driver_name, sizeof(playback_driver_name), "%s", playback_driver_uid);
```

Comment:
from what I know of snprintf usage, if the string is bigger than JACK_CLIENT_NAME_SIZE here, the buffer will not include the last null byte to end the string.
I suggest using `sizeof(...)-1`, then `capture_drive_name[JACK_CLIENT_NAME_SIZE-1] = 0;` afterwards